### PR TITLE
Update some image packages

### PR DESCRIPTION
**Pull request by ad2ien**
Continuing the epic story about improving okp4 dockerhub account, this PR will :
- add some images that are not yet on dockerhub
- give the toolbox image, the description it deserves

### CODE REVIEW

File: dockerfiles/texlive/20220822/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     echo "deb http://deb.debian.org/debian sid main"           \
+     > /etc/apt/sources.list.d/debian-unstable.list             \
+     && apt-get update                                          \  
+     && apt-get install -y --no-install-recommends              \
+     perl-tk=1:804.036-1                                        \
+     wget=1.21-1+deb11u1                                        \
+     xz-utils=5.2.5-2.1~deb11u1                                 \
+     latexmk=1:4.70b-0.2                                        \
+     && apt-get install -y --no-install-recommends -t unstable  \
+     texlive=2022.20220722-1                                    \
```
Comment (amimart) [inline dockerfiles/texlive/20220822/Dockerfile:18]: This dockerfile should be in the `2022.20220722-1` folder

File: dockerfiles/ruby-rdf/3.2.29/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM ruby:3.1.2-alpine3.16
+
+RUN gem install          \
+      rdf:3.2.9 \
```
Comment (amimart) [inline dockerfiles/ruby-rdf/3.2.29/Dockerfile:4]: Should be in the `3.2.9` folder

**Review by amimart — APPROVED** [review]
Nice thank you, I think there is just some imprecisions on folder naming

**Review by amimart — CHANGES_REQUESTED** [review]
Sorry I wrongly approved the changes, please rely on my previous remarks

**Comment by ad2ien** [comment]
> Sorry I wrongly approved the changes, please rely on my previous remarks

Thanks, you're totally right!

**Review by amimart — APPROVED** [review]
🚀

**Pull request merged**